lib/shredder.c

```diff
@@ -1070,7 +1065,7 @@ static void rm_shred_file_preprocess(RmFile *file, RmShredGroup **group) {
 
     /* Create an empty checksum for empty files */
     if(file->file_size == 0) {
-        file->digest = rm_digest_new(cfg->checksum_type, 0, 0, 0, NEEDS_SHADOW_HASH(cfg));
+        file->digest = rm_digest_new(cfg->checksum_type, 0);
```

Comment:
What's being put into the ``checksum`` field of the json output currently?
I couldn't see from a quick look at the diff if the shadow hash is still being calculated.

Reply:
Yes paranoid now *always* has a shadow hash.